jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -171,6 +194,11 @@ public static Configuration fromEnv() {
 
     tracer = getTracerBuilder().build();
     log.info("Initialized tracer={}", tracer);
+
+    if (!(disableGlobalTracer || GlobalTracer.isRegistered())) {
```

Comment:
The same tracer instance can be registered multiple times.

I am wondering whether it would be better to remove `GlobalTracer.isRegistered()` check and propagate potential issues with multiple registered traces at process start time.

Reply:
Oh, you mean that if somebody accidentally calls `fromEnv()` many times, it'll lead to drift between the `GlobalTracer` and the local tracer.
 
I agree, I think it is better to error out. I'll update the code.